internal/server/models/model.go

```diff
@@ -19,7 +19,8 @@ type Model struct {
 	ID uid.ID
 	// CreatedAt is set by GORM to time.Now when a record is first created.
 	// See https://gorm.io/docs/conventions.html#Timestamp-Tracking
-	CreatedAt time.Time
+	// gorm:"<-:create" allows read and create, but not updating
+	CreatedAt time.Time `gorm:"<-:create"`
```

Comment:
This is a side effect of using `Save` as opposed to `Update`. `Save` saves all fields regardless if they've been initialized while `Update` will only save the initialized fields. It's fine to make this change for `CreatedAt` since it only needs to be set once but ideally this is fixed at the source to avoid running into the same problem else where

Reply:
Fair, although our PUT semantics are to replace every field, and this change lines up nicely with the server being the authoritative source for `CreatedAt`. `Updates()` will only replace non-zero fields